backends/platform/dc/selector.cpp

```diff
@@ -156,7 +156,7 @@ static Game the_game;
 static bool isIcon(const Common::FSNode &entry)
 {
   int l = entry.getDisplayName().size();
-  if (l>4 && !strcasecmp(entry.getDisplayName().c_str()+l-4, ".ICO"))
+  if (l>4 && !scumm_stricmp(entry.getDisplayName().c_str()+l-4, ".ICO"))
```

Comment:
The whole body could be replaced with

    return entry.getDisplayName().hasSuffixIgnoreCase(".ICO");

instead.